### ISSUE TITLE
[Bug]: Inkbird IBS-TH2 probe 3 error

### ISSUE THREAD

**Issue by morphiak** [custom-components/ble_monitor#1016]
### What happened?

I got the  IBS-TH2  with external temp probe, works fine when external probe disconnected, errors with

`Inkbird is reporting different probe number. Please report the following data to the developers. data: 0aff8f083f190308224f06
`

when probe connected.
Remedied with insertion of code
```
       elif probe == 3:
            result.update({"temperature probe 1": temp / 100})

```

### Sensor type

IBS-TH2

**Comment by Ernst79** [comment]
So, you are saying that it is reporting as probe number 3? Can you connect more than one probe to the device?

**Comment by morphiak** [comment]
Yup, I printed it out via the logger function. Nope, only one probe, but it's not the common one, it's [this](https://m.media-amazon.com/images/I/619pI6rf+yS._AC_UL640_QL65_.jpg) one.

**Comment by Ernst79** [comment]
Released as 10.3.0
